frontend/packages/dev-console/src/components/pipelineruns/PipelineRunsPage.tsx

```diff
@@ -22,7 +24,9 @@ const PipelineRunsPage: React.FC<PipelineRunsPageProps> = (props) => {
       title={PipelineRunModel.labelPlural}
```

Comment:
Needs to be translated

Reply:
Adding translation to it doesn't affect as it a variable.